app/src/main/java/com/travelbetadisaster/travel_log/ui/journalList/JournalListFragment.kt

```diff
@@ -74,7 +78,8 @@ class JournalListFragment() : Fragment(), OnItemClickListener  {
     }
 
     override fun onItemClick(id: Int) {
-        viewModel.showVisit(id)
+        val action= JournalListFragmentDirections.actionNavJournalListToJournalEntryFragment(id)
+        findNavController().navigate(action)
```

Comment:
I get a null pointer error when I try to click on a list item

``
2024-04-17 01:57:10.617 11905-11905 AndroidRuntime          com.travelbetadisaster.travel_log    E  FATAL EXCEPTION: main
                                                                                                    Process: com.travelbetadisaster.travel_log, PID: 11905
                                                                                                    java.lang.NullPointerException
                                                                                                    	at com.travelbetadisaster.travel_log.ui.journalEntry.JournalEntryFragment.onCreateView(JournalEntryFragment.kt:37)
                                                                                                    	at androidx.fragment.app.Fragment.performCreateView(Fragment.java:3114)
                                                                                                    	at androidx.fragment.app.FragmentStateManager.createView(FragmentStateManager.java:557)
                                                                                                    	at androidx.fragment.app.FragmentStateManager.moveToExpectedState(FragmentStateManager.java:272)
                                                                                                    	at androidx.fragment.app.FragmentManager.executeOpsTogether(FragmentManager.java:1943)
                                                                                                    	at androidx.fragment.app.FragmentManager.removeRedundantOperationsAndExecute(FragmentManager.java:1845)
                                                                                                    	at androidx.fragment.app.FragmentManager.execPendingActions(FragmentManager.java:1782)
                                                                                                    	at androidx.fragment.app.FragmentManager$5.run(FragmentManager.java:565)
                                                                                                    	at android.os.Handler.handleCallback(Handler.java:942)
                                                                                                    	at android.os.Handler.dispatchMessage(Handler.java:99)
                                                                                                    	at android.os.Looper.loopOnce(Looper.java:201)
                                                                                                    	at android.os.Looper.loop(Looper.java:288)
                                                                                                    	at android.app.ActivityThread.main(ActivityThread.java:7872)
                                                                                                    	at java.lang.reflect.Method.invoke(Native Method)
                                                                                                    	at com.android.internal.os.RuntimeInit$MethodAndArgsCaller.run(RuntimeInit.java:548)
                                                                                                    	at com.android.internal.os.ZygoteInit.main(ZygoteInit.java:936)
                                                                                                    	``

Reply:
That has to do with the location ID not being set correctly. It doesn't pull a location and we get a null pointer exception.